__tests__/Ship.test.js

```diff
@@ -0,0 +1,74 @@
+/*global describe it expect */
+const Ship = require('../src/Ship');
+const Port = require('../src/Port');
+const Itinerary = require('../src/Itinerary');
```

Comment:
You can remove most of this here. The comment at the top isnt doing anything so should be removed, and the `Port` and `Itinerary` arent actually used here as youre creating stubs for these in the beforeEach block
```
const Ship = require('../src/Ship');
```